src/lib/components/icons/IconWrapper.svelte

```diff
@@ -1,15 +1,16 @@
 <script lang="ts">
   export let style: string | undefined = undefined;
+  export let fill: string | undefined = undefined;
 </script>
 
 <!-- svelte-ignore a11y-no-static-element-interactions -->
 <svg
   on:click
   on:keydown
-  style={`flex-shrink: 0; ${style ? style : ''}`}
   width="24"
   height="24"
-  fill="var(--color-foreground-level-5)"
+  fill={fill ?? 'var(--color-foreground-level-5)'}
+  style={`flex-shrink: 0; ${style ? style : ''}`}
```

Comment:
Is this order enforced via formatting?

Reply:
No, that change is actually just unnecessary^^ not sure why I did that